go.mod

```diff
@@ -6,20 +6,19 @@ require (
 	github.com/NVIDIA/go-nvml v0.11.6-0
 	github.com/iovisor/gobpf v0.2.0
 	github.com/jszwec/csvutil v1.7.1
-	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.20.2
+	github.com/onsi/ginkgo/v2 v2.4.0
+	github.com/onsi/gomega v1.22.1
 	github.com/prometheus/client_golang v1.13.0
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.37.0
-	golang.org/x/sys v0.0.0-20220829200755-d48e67d00261
+	golang.org/x/sys v0.1.0
 	k8s.io/api v0.25.0
 	k8s.io/klog/v2 v2.70.1
 )
 
 require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/cespare/xxhash/v2 v2.1.2 // indirect
-	github.com/fsnotify/fsnotify v1.4.9 // indirect
```

Comment:
do you know why some pkgs are removed?

Reply:
```
yuanyi@yuanyideMacBook-Pro kepler % go mod why github.com/fsnotify/fsnotify
# github.com/fsnotify/fsnotify
github.com/sustainable-computing-io/kepler/e2e
github.com/sustainable-computing-io/kepler/e2e.test
github.com/onsi/ginkgo
github.com/onsi/ginkgo/internal/remote
github.com/nxadm/tail
github.com/nxadm/tail/watch
github.com/fsnotify/fsnotify
```
above is from ginkgo v1.

```
Switched to branch 'ginkgov2'
yuanyi@yuanyideMacBook-Pro kepler % go mod why github.com/fsnotify/fsnotify
# github.com/fsnotify/fsnotify
(main module does not need package github.com/fsnotify/fsnotify)
```

I suppose in v2 ginkgo removed this package.

Reply:
and I found this commit from ginkgo https://github.com/onsi/ginkgo/commit/102778a4baeb2747386a5b6afa73ebcba5eefe81